src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs

```diff
@@ -55,9 +55,9 @@ public void Setup()
         }
 
         [Test]
-        public void ConstructorWith_BothNullArgs()
+        public void ConstructorWith_AllNullArgs()

```

Comment:
We probably need a test for valid telemetry service, but logger ad auth flows as null?

Reply:
Yes that's true.